components/angular-tomitribe-select/src/tomitribe-select.ts

```diff
@@ -342,6 +343,26 @@ module tomitribe_select {
         }
     }
 
+    function tribeSelectMultipleOpenOnSelect() {
+        return {
+            restrict: 'A',
+            require: 'uiSelect',
+            replace: false,
+            link: link
+        }
+
+        function link($scope, elem, attrs, uiSelect) {
+            if (!uiSelect.multiple) {
+                console.error('tribeSelectMultipleOpenOnSelect should be used along with multiple attribute!');
```

Comment:
Please use $log.error to be angular correct, and probably return in this place I don't see a reason to have this attribute on other ui-selects